fixtures/golden/codegen/go/chi/mysql/url_shortener/.env.example

```
@@ -2,6 +2,7 @@ DATABASE_URL=url_shortener:url_shortener@tcp(db:3306)/url_shortener?parseTime=tr
 BASE_URL=http://localhost:8080
 PORT=8080
 LOG_LEVEL=info
+ADMIN_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_TOKEN key should go before the BASE_URL key

(UnorderedKey)